index.php

```diff
@@ -54,7 +53,7 @@
                     ?>
                 </div>
                 <button type="submit" id="download" class="suggested-action">Download Freya</button>
-                <p class="small-label"><?php echo round($filesize / 1000 / 1000); ?> MB (for PC or Mac)</p>
+                <p class="small-label">893 MB (for PC or Mac)</p>
```

Comment:
@cassidyjames We have to manually update the links anyway, so I don't see this as a major problem.